tests/linearizability/model.go

```diff
@@ -57,15 +57,15 @@ type ValueRevision struct {
 }
 
 var etcdModel = porcupine.Model{
-	Init: func() interface{} { return "{}" },
+	Init: func() interface{} { return "[]" },
```

Comment:
```suggestion
	Init: func() interface{} { return "{}" },
```

Reply:
I changed storing `etcdState` to storing `[]etcdState`. Are you saying it's a bad change or don't understand the purpose?